config/jobs/kubernetes-csi/gen-jobs.sh

```diff
@@ -25,7 +25,7 @@ latest_stable_k8s_version="1.17.0"
 latest_stable_k8s_minor_version="1.17"
 
 # We need this image because it has Docker in Docker and go.
-dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191213-55437e3-master"
+dind_image="gcr.io/k8s-testimages/kubekins-e2e:v20191231-77c7890-master"
```

Comment:
Can you change this back to v20191221-fe232fc-master? There should be a bot that updates the images when a new version is available and validated.